### PR TITLE
Feat PUD-972 (pci.storage.databases): add replication tab

### PR DESCRIPTION
**Pull request by jperchoc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/PUD-819`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | PUD-972
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] ~~Standalone app was ran and tested locally~~ (not applicable)
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ (not applicable)

## Description

Add replication tab for new engine

## Related
```
PUD-833 <-- PUD-819
               ^
               |-- PUD-975 <-- PUD-972 
				^
				|-- PUD-969
				|-- PUD-973 -- PUD-1046 -- PUD-1057 -- PUD-1047
				|-- PUD-1007
```
[PUD-833](https://github.com/ovh/manager/pull/5605)
[PUD-819](https://github.com/ovh/manager/pull/5747)
[PUD-975](https://github.com/ovh/manager/pull/5765)
[PUD-972](https://github.com/ovh/manager/pull/5748)
[PUD-969](https://github.com/ovh/manager/pull/5774)
[PUD-973](https://github.com/ovh/manager/pull/5749)
[PUD-1007](https://github.com/ovh/manager/pull/5769)
[PUD-1046](https://github.com/ovh/manager/pull/5865)
[PUD-1057](https://github.com/ovh/manager/pull/5872)
[PUD-1047](https://github.com/ovh/manager/pull/5873)

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/replications.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            data-filterable
+        >
+            <div
+                class="d-inline"
+                data-ng-repeat="topic in $row.topics track by $index"
+            >
+                <br
+                    data-ng-if="$row.topics.length > $ctrl.MAX_TOPIC_PER_LINE && $index % $ctrl.MAX_TOPIC_PER_LINE === 0"
+                />
+                <span class="oui-badge oui-badge_info">{{topic}}</span>
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/replications.html:57]:
```suggestion
                <span class="oui-badge oui-badge_info" data-ng-bind="topic"></span>
```

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/replications.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            data-filterable
+        >
+            <div
+                class="d-inline"
+                data-ng-repeat="topic in $row.topicExcludeList track by $index"
+            >
+                <br
+                    data-ng-if="$row.topicExcludeList.length > $ctrl.MAX_TOPIC_PER_LINE && $index % $ctrl.MAX_TOPIC_PER_LINE === 0"
+                />
+                <span class="oui-badge oui-badge_info">{{topic}}</span>
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/replications.html:75]:
```suggestion
                <span class="oui-badge oui-badge_info" data-ng-bind="topic"></span>
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/replications.html (hunk 1 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/replications.html]: Done: https://github.com/ovh/manager/pull/5748/commits/70695694870fc8959de807632647c2392dbd1b88

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/replications.html (hunk 2 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/replications.html]: Done: https://github.com/ovh/manager/pull/5748/commits/70695694870fc8959de807632647c2392dbd1b88

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "pci_databases_replications_tab_title": "Flux de réplication",
+  "pci_databases_replications_tab_description": "Les flux de réplication permettent la copie automatique des topics de données et de la configuration entre deux clusters Kafka. Un service MirrorMaker peut exécuter plusieurs flux de réplication.",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/translations/Messages_fr_FR.json:3]:
```suggestion
  "pci_databases_replications_tab_description": "Les flux de réplication permettent la copie automatique des topics de données et de la configuration entre deux clusters Kafka. Un service Mirror Maker peut exécuter plusieurs flux de réplication.",
```

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -57,5 +57,7 @@
   "pci_databases_general_information_certificate_label": "Certificat CA",
   "pci_databases_general_information_certificate_link": "Voir",
   "pci_databases_general_information_certificate_download": "Télécharger",
-  "pci_databases_general_information_certificate_tooltip": "Votre certificat: <code>{{certificate}}</code>"
+  "pci_databases_general_information_service_integration_title": "Intégration de services",
+  "pci_databases_general_information__kafka_mirror_maker_label": "Kafka Mirror Maker",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json:61]:
```suggestion
  "pci_databases_general_information_kafka_mirror_maker_label": "Kafka Mirror Maker",
```

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -57,5 +57,7 @@
...
+  "pci_databases_general_information__kafka_mirror_maker_none": "Aucun"
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json:62]:
```suggestion
  "pci_databases_general_information_kafka_mirror_maker_label": "Kafka Mirror Maker",
  "pci_databases_general_information_kafka_mirror_maker_none": "Aucun"
```

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/translations/Messages_fr_FR.json (hunk 3 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/translations/Messages_fr_FR.json:3]: Probably, but as said before, I have to use the exact text given by content team.

File: packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json (hunk 5 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json]: Good catch ! https://github.com/ovh/manager/pull/5748/commits/cfd544f2fdffc49f4273fd20f12853325a059ead

**Review by radireddy — APPROVED** [review]

**Comment by mohammed-zahaf** [comment]
Good or me @jperchoc.
Please don't forget to add my content suggesstions into Jira ticket for QA check.

Thanks.

**Review by mohammed-zahaf — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**